platform/security/docs/src/main/resources/IntContents.adoc

```diff
@@ -1426,3 +1426,58 @@ The `AnonymousInterceptor` is installed by default with DDF Security Application
 . Click save.
 
 Once these configurations have been added, the AnonymousInterceptor is ready for use. Both secure and non-secure requests will be accepted by all secure DDF service endpoints.
+
+== Security IdP
+
+The Security IdP application provides service provider handling that satisfies the http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf[SAML 2.0 Web SSO profile] in order to support external IdPs (Identity Providers).
+
+=== Components
+[cols="3", options="header"]
+|===
+
+|Bundle Name
+|Located in Feature
+|Description
+
+|security-idp-sp
+|security-idp
+|IdP Service Provider
+
+|security-idp-server
+|security-idp
+|IdP Server
+
+|===
+
+=== Installation
+
+These bundles are not installed by default, they can be started by installing the `security-idp` feature.
+
+=== Security IdP Service Provider
+
+The IdP client that interacts with the specified Identity Provider.
+
+==== Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> IdP Client
```

Comment:
These arrows only show up in the HTML docs and not the PDF docs for some reason. Not sure if it's worth changing them or not?